cmd/security-agent/app/app.go

```diff
@@ -161,6 +162,11 @@ func start(cmd *cobra.Command, args []string) error {
 	// Check if we have at least one component to start based on config
 	if !coreconfig.Datadog.GetBool("compliance_config.enabled") && !coreconfig.Datadog.GetBool("runtime_security_config.enabled") {
 		log.Infof("All security-agent components are deactivated, exiting")
+
+		// a sleep is necessary to ensure that supervisor registers this process as "STARTED"
+		// If the exit is "too quick", we enter a BACKOFF->FATAL loop even though this is an expected exit
+		// http://supervisord.org/subprocess.html#process-states
```

Comment:
nit: this comment is outdated (only applied to Agent v5), nowadays this is still true but only for the sysvinit scripts that we package for Debian 7 and SUSE 11